### PR TITLE
Stat screen and incorrect songs screen

### PR DESCRIPTION
**Pull request by Tsathogguaa**
Added two activies to show users how they performed in a game:

image
image

### CODE REVIEW

**Review by laurislopata — APPROVED** [review]
Some refactoring is needed to fix codeclimate issues but we can refactor these as next weeks tasks

File: app/src/main/java/ch/sdp/vibester/IncorrectSongsScreen.kt [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.Gravity
+import android.view.View
+import android.widget.LinearLayout
+import android.widget.LinearLayout.*
+import android.widget.TextView
+
+class IncorrectSongsScreen : AppCompatActivity() {
```
Comment (MaximeZmt) [inline app/src/main/java/ch/sdp/vibester/IncorrectSongsScreen.kt:12]: Doc would be great

File: app/src/main/java/ch/sdp/vibester/GameEndingScreen.kt [hunk 2]
```diff
@@ -0,0 +1,75 @@
+package ch.sdp.vibester
+
+import androidx.appcompat.app.AppCompatActivity
+import android.content.Intent
+import android.os.Bundle
+import android.view.View
+import android.widget.*
+
+class GameEndingScreen : AppCompatActivity() {
```
Comment (MaximeZmt) [inline app/src/main/java/ch/sdp/vibester/GameEndingScreen.kt:9]: doc too ?

File: app/src/main/java/ch/sdp/vibester/GameEndingScreen.kt [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+        // filling the values here.
+        val stat1: TextView = findViewById(R.id.end_stat1)
+        stat1.text = stat1name //stat1.text = statNames[0]
+        val stat1res: TextView = findViewById(R.id.end_stat1_res)
+        stat1res.text = stat1resName //stat1res.text = statValues[0]
+        //--------------- Example setup ---------------
+
+
+        val playerNameView: TextView = findViewById(R.id.end_player_name)
+        val statPlayerText = "Here are the stats for the player $playerName"
```
Comment (MaximeZmt) [inline app/src/main/java/ch/sdp/vibester/GameEndingScreen.kt:59]: hard coded text

**Review by MaximeZmt — APPROVED** [review]
Approved! but you should really add the doc soon ! (and clean all commented code)

**Pull request merged**